File: src/main/java/com/campool/CampoolApplication.java
package com.campool;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cache.annotation.EnableCaching;
import org.springframework.session.data.redis.config.annotation.web.http.EnableRedisHttpSession;

@EnableCaching
@EnableRedisHttpSession
@SpringBootApplication
public class CampoolApplication {

    public static void main(String[] args) {
        SpringApplication.run(CampoolApplication.class, args);
    }

}

File: src/main/java/com/campool/CampoolApplication.kt
package com.campool

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cache.annotation.EnableCaching

@EnableCaching
@SpringBootApplication
class CampoolApplication

fun main(args: Array<String>) {
    runApplication<CampoolApplication>(*args)
}

